src/assets/scripts/client/ui/TutorialView.js

```diff
@@ -490,33 +477,10 @@ export default class TutorialView {
 
         this.tutorial_step({
             title: 'Approach Clearances, part 2',
-            text: ['You may choose to enter one command at a time, but air traffic controllers usually do multiple. Particularly in approach clearances,',
-                   'they follow an acronym "PTAC" for the four elements of an approach clearance, the "T" and "C" of which',
-                   'stand for "Turn" and "Clearance", both of which we entered separately in this tutorial. Though longer, it is both ',
-                   'easier and more real-world accurate to enter them together, like this: "fh 250 i 28r".'
-            ].join(' '),
-            side: 'left',
-            position: tutorial_position
-        });
-
-        this.tutorial_step({
-            // TODO: the windsock will be moving soon. Update this when that happens
-            title: 'Wind sock',
-            text: ['In the lower right corner of the map is a small circle with a line. You may need to collapse the',
-                   'the flight strips with the \'>|\' tab. It\'s like a flag: the line trails in the direction',
-                   'the wind is blowing toward. If it\'s pointing straight down, the wind is blowing from the North',
-                   'to the South. Aircraft must be assigned to different runways such that they always take off and land into the wind, unless the',
-                   'wind is less than 5 knots.'
-            ].join(' '),
-            side: 'left',
-            position: tutorial_position
-        });
-
-        this.tutorial_step({
-            title: 'Scope Commands',
-            text: ['There are also various commands that can be entered into your "scope" which deal with moving',
-                   'aircraft data blocks (labels), transferring control of aircraft, etc. To toggle between aircraft',
-                   'commands and scope commands, press the tab key.'
+            text: ['As said earlier, in the Arrivals page, before clearing the aircraft for the ILS, it should be at an appropriate',
+                   'altitude and a heading of 30 degrees or less to intercept. For maximum realism, we recommend looking up the IAP',
+                   'charts for the airport you are at, to find the intercept altitude and any other restrictions. You can find these',
+                   'easily via a quick google search! :)'
             ].join(' '),
             side: 'left',
             position: tutorial_position
```

Comment:
The score section should say "it doesn't have any effect *on* the simulation"

Reply:
Something I overlooked again :)